2_task(magazine)/js/dropdown.js

```diff
@@ -0,0 +1,25 @@
+function myFunction(elem) {
+  document.getElementById(elem.getAttribute("data-name")).classList.toggle("show");
+}
+
+window.onclick = function(event) {
+  var obj = event.target;
+  let dropdowns = document.getElementsByClassName("drop");
+  if (!obj.hasAttribute("data-name")) {
```

Comment:
data attr i not very specific, I can use the same attr for another elements. data-dropdown-menu could be used or smth like that